src/main/java/com/example/java/test/junior/developer/dto/LoginRequestDto.java

```diff
@@ -0,0 +1,23 @@
+package com.example.java.test.junior.developer.dto;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+
+@Data
+@Builder
+@NoArgsConstructor(force = true)
+@AllArgsConstructor
+public class LoginRequestDto {
+
+  @Email(message = "Invalid email format")
```

Comment:
`@Email` annotation will not validate email by its own, unfortunately, you need to provide regex. Check @aleksdenni examples.